WordPress/wordpress-playground
# Use NPM for publishing packages instead of Lerna

## Motivation for the change, related issues

Every few months there's a problem with publishing packages via Lerna, e.g. here's the latest one:

https://github.com/WordPress/wordpress-playground/issues/1572

Debugging those issues takes a lot of time and typically there are no useful resources out there.

This PR limits the use of Lerna to bumping the packages version in the repo and tagging the updated code.

## Implementation details

Instead of using `lerna publish`, we run a simple, custom bash script.

## Testing Instructions (or ideally a Blueprint)

Merge and run. It seems to be working locally and I don't see any other way of testing it in GitHub.
